roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
-- name: setup swift rings
-  command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
+- name: setup swift rings - source
+  command: sudo -u swiftops {{ openstack_source.virtualenv_base }}/swift/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
 
+- name: setup swift rings - package
+  command: sudo -u swiftops {{ openstack_package.virtualenv_base }}/swift/bin/swifttool -i /home/swiftops/.ssh/id_rsa
+           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
```

Comment:
Having two handlers here shouldn't be necessary if you use the alternatives system to expose swiftool from the venv into the regular path.